pdns/iputils.hh

```diff
@@ -261,6 +261,16 @@ union ComboAddress {
     return ret;
   }
 
+  bool isValid() const
+  {
+    char host[1024];
+    int retval = 0;
+    if(sin4.sin_family && !(retval = getnameinfo(reinterpret_cast<const struct sockaddr*>(this), getSocklen(), host, sizeof(host),0, 0, NI_NUMERICHOST)))
```

Comment:
I'm not sure this check is actually useful?

Reply:
I also wondered. This question stayed open in the original review, which I totally failed to reread before asking for your review

Reply:
I removed isValid entirely